fvtest/porttest/omrsockTest.cpp

```diff
@@ -578,3 +578,91 @@ TEST(PortSockTest, two_socket_datagram_communication)
 	EXPECT_EQ(OMRPORTLIB->sock_close(OMRPORTLIB, &clientSocket), 0);
 	EXPECT_EQ(OMRPORTLIB->sock_close(OMRPORTLIB, &serverSocket), 0);
 }
+
+/**
+ * Equal operator to compare contents of OMRTimeval struct socket option values.
+ */
+bool operator==(const OMRTimeval& lhs, const OMRTimeval& rhs) 
+{
+	if ((lhs.data.tv_sec != rhs.data.tv_sec) || (lhs.data.tv_usec != rhs.data.tv_usec)) {
+		return false;
+	}
+	return true;
+}
+
+/**
+ * Equal operator to compare contents of OMRLinger struct socket option values.
+ */
+bool operator==(const OMRLinger& lhs, const OMRLinger& rhs) 
+{
+	if ((lhs.data.l_onoff != rhs.data.l_onoff) || (lhs.data.l_linger != rhs.data.l_linger)) {
```

Comment:
This can be simplified to a single return statement:
```suggestion
	return (lhs.data.l_onoff == rhs.data.l_onoff) && (lhs.data.l_linger == rhs.data.l_linger);
```